toycar:
  FrontLeftWheelJoint:
    has_positon_limits: true
    min_position: -0.5
    max_position: 0.5
    has_effort_limits: true
    max_effort: 3.0
  FrontRightWheelJoint:
    has_positon_limits: true
    min_position: -0.5
    max_position: 0.5
    has_effort_limits: true
    max_effort: 3.0
  left_frontaxisjoint:
    has_positon_limits: false
    has_velocity_limits: true
    max_velocity: 5.0
    has_effort_limits: true
    max_effort: 3.0
  right_frontaxisjoint:
    has_positon_limits: false
    has_velocity_limits: true
    max_velocity: 5.0
    has_effort_limits: true
    max_effort: 3.0